docs/arch.plantuml

```diff
@@ -0,0 +1,38 @@
+@startuml
+center header <b>Ceres Rollup Service Architecture
+queue "unified.metrics.json topic" as input
+card Metrics
+note right: com.rackspacecloud.metrics.rollup.domain.Metric
+node KTable [
+<b> KTable
+==========
+5m Tumbling Window
+==========
+20m Tumbling Window
+==========
+60m Tumbling Window
+==========
+240m Tumbling Window
+==========
+1440m Tumbling Window
+] 
+note right: com.rackspacecloud.metrics.rollup.config.TopologyConfiguration
+card Rollups
+note right: com.rackspacecloud.metrics.rollup.domain.RolledUp
```

Comment:
...and similarly, I would have expected the use of ExternalMetric but with special field suffixes, like "_max", "_avg".